layers/meta-balena-jetson/recipes-kernel/linux/jetson-dtbs.bb

```diff
@@ -9,6 +9,12 @@ do_install[depends] += " nvidia-kernel-oot:do_deploy "
 
 SRC_URI += " file://tegra234-p3737-0000+p3701-0000-nv-spi.dtb "
 
+SRC_URI:append:forecr-dsb-ornx = " \
+    file://forecr-dsb-ornx/tegra234-p3768-0000+p3767-0000-nv.dtb \
```

Comment:
@asuykov could you provide a link to the BSP from where these device-trees were obtained?

Reply:
Already there, linux-jammy-nvidia-tegra/forecr-dsb-ornx/README.